Sources/OpenAPIKit/Schema Object/JSONSchemaContext.swift

```diff
@@ -598,9 +598,10 @@ extension JSONSchema {
         ///     is determined by looking at its properties'
         ///     required Bool.
         public var requiredProperties: [String] {
-            return Array(properties.filter { (_, schemaObject) in
-                schemaObject.required
-            }.keys).sorted()
+            properties
+                .filter { _, schema in schema.required }
+                .map { $0.key }
+                .sorted()
```

Comment:
It might make more sense to drop this sorting now. I think I added it in the first place because when properties was an unsorted dictionary it was not desirable that the required properties array would be encoded with different ordering each time (this instability made diffing appear to indicate changes had been made when they had not). 

Anyway, since this is backed by an ordered dictionary now, we can probably get stable order without sorting which may be preferable anyway.

Reply:
Removed sorting.